scripts/install_ebpf.psm1

```diff
@@ -67,6 +67,10 @@ function Register-eBPFComponents
                 Write-Log ("{0} driver created." -f $_.Name) -ForegroundColor Green
             }
         }
+        # Enable KMDF verifier and tag tracking.
+        New-Item -Path ("HKLM:\System\CurrentControlSet\Services\{0}\Parameters\Wdf" -f $_.Name) -Force -ErrorAction Stop
+        New-ItemProperty -Path ("HKLM:\System\CurrentControlSet\Services\{0}\Parameters\Wdf" -f $_.Name) -Name "VerifierOn" -Value 1 -PropertyType DWord -Force -ErrorAction Stop
+        New-ItemProperty -Path ("HKLM:\System\CurrentControlSet\Services\{0}\Parameters\Wdf" -f $_.Name) -Name "TrackHandles" -Value "*" -PropertyType MultiString -Force  -ErrorAction Stop
```

Comment:
While we want this enabled in CI/CD and when installing for testing, we probably don't want this on for a simple runtime install on normal runtime machines.